### PR TITLE
adding vmss sample

### PR DESCRIPTION
**Pull request by zikalino**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: vmss/get-hosts-tasks.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+- name: Get facts for all Public IPs within a resource groups
```
Comment (yungezz) [inline vmss/get-hosts-tasks.yml]: pls add description to all samples, and dependencies, such as role name

File: vmss/vmss-create.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        name: "{{ vmss_name }}"
+        rules:
+          - name: SSH
+            protocol: Tcp
+            destination_port_range: 22
+            access: Allow
+            priority: 1001
+            direction: Inbound
+
+    #- name: Create a load balancer
```
Comment (yungezz) [inline vmss/vmss-create.yml]: why not load balaner?

File: vmss/vmss-create.yml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    #    natpool_frontend_port_end: 50040
+    #    natpool_backend_port: 22
+    #    natpool_protocol: Tcp
+
+    - name: Create VMSS
+      azure_rm_virtualmachine_scaleset:
+        resource_group: "{{ resource_group }}"
+        name: "{{ vmss_name }}"
+        vm_size: Standard_DS1_v2
+        admin_username: zim
```
Comment (yungezz) [inline vmss/vmss-create.yml]: put credentials in vars via lookup?

File: vmss/vmss-dump-using-module.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (yungezz) [inline vmss/vmss-dump-using-module.yml]: the naming is confusing. it's hard to understand what's different between this one and vmss_dump.yml. I suggest you to add a readme file to describe what  you want to say by adding those yaml files.

Comment (zikalino) [reply vmss/vmss-dump-using-module.yml]: maybe some playbooks are not necessary, that's another issue

**Pull request merged**